### PR TITLE
Issue/55 nabu

### PR DESCRIPTION
**Pull request by iperdomo**
This cron job uses Lumen API to synchronize data from Flow to a PostGIS database: 
`Flow -> Lumen -> CronJob -> PostGIS`

* There are some Dockerfiles related to the latest version of PostgreSQL 12 and PostGIS 3.0
* Currently all hosted solutions (Amazon RDS, Google Cloud SQL, others?) offer PostgreSQL 11 and PostGIS 2.5
* The Python code requires `requests` and `psycopg2` libraries. The interesting files are `pg.py` and `update.py`
* The code expects that the database exists and that have the extensions ready: `postgis` `btree_gist`
* Based on the data structure provided by Lumen (columns and rows), we create an new table in the `import` schema, and then update the _view_ on the `public` schema. NOTE: Currently, we don't remove previous imports
* When connecting from an external system, is useful to only list the objects in the `public` schema (default one)
* We create additional views _per location_, so the user can pick the view according to the top level location answer (a _cascade_ question)

NOTES: 
* After merging we need to modify `deploy.sh` to enable the check to only deploy on `develop` or `master` branch

### CODE REVIEW

File: lumen-api/Jobs_Nabu_PostGIS/ci/k8s/cronjob.yaml.template [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: nabu-postgis
 spec:
-  schedule: "* */2 * * *"
+  schedule: "@hourly"
```
Comment (dlebrero) [inline lumen-api/Jobs_Nabu_PostGIS/ci/k8s/cronjob.yaml.template]: Given an hourly schedule, with an allow concurrencyPolicy, a backoffLimit of 6 and an activeDeadlineSeconds of 15 mins, it means that it maybe two jobs will end up running at the same time. Is that ok?

Comment (joycarpediem) [reply lumen-api/Jobs_Nabu_PostGIS/ci/k8s/cronjob.yaml.template]: Given the velocity of data i think for starters , 0 0 * * 0 will suffice

Comment (iperdomo) [reply lumen-api/Jobs_Nabu_PostGIS/ci/k8s/cronjob.yaml.template]: @dlebrero my code was a _skeleton_ I didn't have a the actual business requirement in how often the data needs to be refreshed. I wanted to test that the code/cronjob works with frequent updates.

**Comment by iperdomo** [comment]
@joycarpediem Cron definition updated, the monorepo-build.sh script is failing on the PR build, but I think we're good to go:
```
fatal: ambiguous argument 'FETCH_HEAD..develop': unknown revision or path not in the working tree.
```

**Pull request merged**